### DISCUSSION TITLE
Periodically update camera entity picture

### DISCUSSION THREAD

**Discussion by surskitt** [fuatakgun/eufy_security#178] [General]
I would prefer to have the picture on the camera entity updated to show a somewhat recent image rather than the last time someone went into the garden...which could either be days ago, wrong time of day, or both.

Does anyone have any ideas on how this could be done.

One thing I'm thinking is have a periodic automation turn the camera on, take a snapshot to a directory, then turn the camera off. Then I could have the [Refreshable picture card](https://github.com/dimagoltsman/refreshable-picture-card) pick up that picture on a shorter interval than the automation. I've not tried this yet since I've had trouble even getting the automated snapshots to work, since sometimes the camera does not turn on and even when it does, sometimes it doesn't turn off.

Does anyone else have any other suggestions for how this could be done?

**Comment by surskitt** [comment]
Actually, there's no need for any special card, just making a local_file camera is fine. Now if only I could get the snapshots taken reliably, the camera doesn't always want to start...

**Reply by fuatakgun** [reply]
there is no solution to get the snapshot without starting to live stream and you need to save the snapshot after starting the camera. If you have any issues while starting the camera, please create an issue

**Comment by moishap** [comment]
I am also looking for the easiest automated way to take periodic snapshots

**Reply by fuatakgun** [reply]
There is an automation example on readme, works for me.

**Reply by moishap** [reply]
Yeah, not for me though ...

**Comment by shanelord01** [comment]
Try my automation I just posted? https://github.com/fuatakgun/eufy_security/discussions/789#discussion-5135307

**Comment by bkr1969** [comment]
Where does the snapshot go?  I still have a weeks old picture for one camera and the one that somewhat seems to be working is a really zoomed in image rather than the entire field of view. I think that image is only being generated with a motion event anyway, not a snapshot.
